meson_options.txt

```diff
@@ -27,6 +27,7 @@ option('plugin_emmc', type : 'feature', description : 'eMMC support', deprecated
 option('plugin_ep963x', type : 'feature', description : 'EP963x support', deprecated: {'true': 'enabled', 'false': 'disabled'})
 option('plugin_fastboot', type : 'feature', description : 'Fastboot support', deprecated: {'true': 'enabled', 'false': 'disabled'})
 option('plugin_gpio', type : 'feature', description : 'Linux GPIO support', deprecated: {'true': 'enabled', 'false': 'disabled'})
+option('plugin_igsc', type : 'feature', description : 'Intel Graphics System Controller support')
```

Comment:
any reason not to just call this graphics?

Reply:
I think Intel wanted to make it clear that existing Intel graphics wasn't supported, only the ones in libigsc

Reply:
Libigsc? I guess that's what this code comes from?

Reply:
Indeed.